### PR TITLE
I implemented tests for some of the functions

### PR DESCRIPTION
**Pull request by caganze**
This is a first iteration for implementing tests. For now, everything passes.

### CODE REVIEW

File: tests/test_functions.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+	stack = np.stack((img1,img2))
+	assert (combflagstack(stack) == np.array([[1, 2, 0], [3, 0, 4], [0, 0, 0]])).all()
+
+def test_dictaddentry():
+    dicta = {'HA':1,'PA':2,'MJD': 3}
+    res=dictaddentry(dicta,'MJD','before','new',4)
+    assert res['HA']== 1
+    assert res['PA']== 2
+    assert res['new']== 4
+    assert res['MJD']== 3
```
Comment (kelle) [inline tests/test_functions.py:55]:
```suggestion
    res = dictaddentry(dicta, 'MJD', 'before', 'new', 4)
    assert res['HA'] == 1
    assert res['PA'] == 2
    assert res['new'] == 4
    assert res['MJD']== 3
```

File: tests/test_functions.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+def test_sixty():
+	x = -0.0111234
+	assert (sixty(x)==np.array([0.0, 0.0, -40.04424])).all()
+
+	assert (sixty(x,trailsign=True)==np.array([-0.0, 0.0, 40.04424])).all()
+
+	assert (sixty(x,colons={'dec':2,'plus':1},trailsign=True)=='-00:00:40.04')
+
+	x = 0.0111234
+	assert (sixty(x,colons={'dec':2,'plus':1},trailsign=True)=='+00:00:40.04')
```
Comment (kelle) [inline tests/test_functions.py:127]: I think we should also test these against the astropy implementations. https://docs.astropy.org/en/stable/coordinates/index.html#example

File: tests/test_functions.py [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+
+def test_polyfit1d():
+	x=  np.array([10. ,20. ,30. ,40. ,50., 60. ,70. ,80. ,90.,100.])
+	y=  np.array([0.37,0.58,0.83,1.15,1.36,1.62,1.90,2.18,2.45,np.nan])
+	yerr = np.array([0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05,0.05])
+	res=  polyfit1d(x,y,1,yunc=yerr)
+	pass
+	#will implement test later
```
Comment (kelle) [inline tests/test_functions.py:136]: make an `xfail`

**Review by kelle — CHANGES_REQUESTED** [review]
- As we discussed, move tests to be in files which correspond to their submodule (eg., `test_fitsutils`.
- use `assert = 0` to make tests fail as expected instead of `pass`.

File: tests/test_functions.py [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+def test_bicucof():
+	pass
+
+@xfail(reason='examples not implemented')
+def test_bicuval():
+	pass
+
+@xfail(reason='examples not implemented')
+def test_findorders():
+	pass
```
Comment (kelle) [inline tests/test_functions.py:30]: change `pass` to `assert = 0`

File: tests/test_functions.py (hunk 1 above)
Comment (kelle) [reply tests/test_functions.py:55]: just adding some PEP8 spaces

File: tests/test_functions.py (hunk 4 above)
Comment (caganze) [reply tests/test_functions.py]: I have changed from pass to assert= 0 and I have changed from xfail to skip

File: tests/test_functions.py (hunk 2 above)
Comment (caganze) [reply tests/test_functions.py]: I will try this in the next iteration

**Pull request closed without merging**